v_4_0_0/cloudconfig.go

```diff
@@ -54,14 +54,6 @@ func NewCloudConfig(config CloudConfigConfig) (*CloudConfig, error) {
 		return nil, microerror.Maskf(invalidConfigError, "config.Template must not be empty")
 	}
 
-	// Default to 443 for non AWS providers.
-	if config.Params.EtcdPort == 0 {
-		config.Params.EtcdPort = 443
-	}
-
-	// extract cluster base domain
-	config.Params.BaseDomain = strings.TrimPrefix(config.Params.Cluster.Kubernetes.API.Domain, "api.")
```

Comment:
This part is moved into operator (e.g. https://github.com/giantswarm/kvm-operator/blob/ignition/service/controller/v18/cloudconfig/master_template.go#L23) as it is required for rending of the templates